sklearn/linear_model/_base.py

```diff
@@ -419,6 +412,27 @@ class LinearRegression(MultiOutputMixin, RegressorMixin, LinearModel):
         ``-1`` means using all processors. See :term:`Glossary <n_jobs>`
         for more details.
 
+    normalize : bool, default=False
+        This parameter is ignored when ``fit_intercept`` is set to False.
+        If True, the regressors X will be normalized before regression by
+        subtracting the mean and dividing by the l2-norm.
+        If you wish to standardize, please use
+        :class:`sklearn.preprocessing.StandardScaler` before calling ``fit`` on
+        an estimator with ``normalize=False``.
+
+    positive : bool, default=False
+        When set to ``True``, forces the coefficients to be positive. This
+        option is only supported for dense arrays.
+
+        .. versionadded:: 0.24
+           LinearRegression might force the coefficients to be positive.
+
+    maxiter : int, default=None
```

Comment:
should be max_iter

Reply:
I have used max_iter [here](https://github.com/scikit-learn/scikit-learn/pull/17578/commits/6a40d90979a23e7380abb437b8176ef23d61e589) and common tests failed looking for an `n_iter` attribute I hadn't defined: there is probably a clever way to avoid this, but I'm wondering if it is worth it to add a max_iter/maxiter parameter ... back compatibility with scipy 0.19.1 is not guarantee (see [comment](https://github.com/scikit-learn/scikit-learn/pull/17578#discussion_r440020098))... unless I am missing something... I need some direction in that case... Thanks!

Reply:
we're using `max_iter` in (most of) the rest of the code base so we should use it here too ;)

We have tests that make sure than when the `max_iter` parameter exists, there should also be a `n_iter_` attribute. These are `check_non_transformer_estimators_n_iter` and `check_transformer_n_iter` (maybe others?). If we can expose `n_iter_` then we should do it, otherwise both these checks have a hardcoded list of estimators that they ignore so we will probably have to add `LinearRegression` there.

Reply:
Thanks for your answer, this solves my first issue. I will modified maxiter -> max_iter once the back compatibility issue is clarified too. Thanks for your patience.